src/renderer/pages/Home.js

```diff
@@ -131,6 +159,26 @@ const Home = observer(() => {
           SshKey={currentKey}
         />
       ) : null}
+
+      {showInputNameDialog ? (
+        <InputDialog
+          title="Add Username"
+          placeholder="Enter username"
```

Comment:
Shouldn't we add placeholder like `Enter your github's username" to make it even more clear?

Reply:
Makes sense. Done!